src/main/java/ch/njol/skript/effects/EffRemoveResourcePack.java

```diff
@@ -0,0 +1,84 @@
+package ch.njol.skript.effects;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Effect;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.util.Utils;
+import ch.njol.util.Kleenean;
+import org.bukkit.entity.Player;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.UUID;
+
+@Name("Remove Resource Pack")
+@Description({"Remove all resource packs or a resource pack by uuid sent by the server.",
+	"",
+	"You can allow up to 32 alphanumeric characters in the uuid."
+})
+@Examples({
+	"remove all resource packs from all players",
+	"remove resource pack with the uuid \"1\" from all players"
+})
+@Since("INSERT VERSION")
+public class EffRemoveResourcePack extends Effect {
+
+	static {
+		Skript.registerEffect(EffRemoveResourcePack.class, "remove [:all] resource pack[s] [id:with [the] [uu]id %string%] from %players%");
```

Comment:
this has been an ongoing thing I've seen but `id:` is not required, if the `%string%` is null the `with uuid` has not used